### ISSUE TITLE
[feat] macOS .app.tar.gz and .app.tar.gz.sig should include app version number 

### ISSUE THREAD

**Issue by goenning** [tauri-apps/tauri#5890]
### Describe the problem

I always have to rename macOS bundles when releasing my app. On windows and linux, all bundles start with `AppName_Version_***`, while in macOS it's just `AppName_***`

And this naming is required if you're pushing assets to S3 for the auto-uploader to pick up, as you need distinct names for each version.

### Describe the solution you'd like

Name macOS bundles just like the other platforms.

### Alternatives considered

I can rename it myself, manually or scripted, which is what I do today, but this would be that nice extra DX touch if Tauri would do it automatically :)

**Comment by goenning** [comment]
Note: the .dmg is named correctly, it's only the `.app.tar.*` that are not.

**Comment by lucasfernog** [comment]
I'll take a look at this again but last time we tried it, Apple was failing to sign/notarize the file if it had the version number.

**Comment by goenning** [comment]
What if we rename it after the sign/notarize?

**Comment by goenning** [comment]
well actually, I believe what you sign/notarize is the .app? And then after that it gets tar zipped right? If that's the case, then it should be ok because we don't really need the .app during the update, only the tarzip and sig

**Comment by lucasfernog** [comment]
Yeah you're right, and sign/notarization is also working anyway. But now this is a breaking change that would need to wait for v2 :(

**Comment by AlexanderBastowros** [comment]
Is this issue closable?
